browser/ui/views/brave_actions/brave_actions_container.h

```diff
@@ -187,6 +190,8 @@ class BraveActionsContainer : public views::View,
 
   std::unique_ptr<EmptyExtensionsContainer> empty_extensions_container_;
 
+  OnionLocationView* onion_location_view_ = nullptr;
```

Comment:
I think onion view is not brave action.
IMO, OnionLocationView needs another parent view. WDYT?

Reply:
@petemill do we need another parent for it? Or should we rename `BraveActionContainer` to something that is designated for Brave Icons?

Reply:
Well, I'm leaning towards suggesting we put this in the regular `PageAction` area (or simply in the `LocationBar` directly and not the `BraveActionsContainer`. `BraveActionsContainer` is so named because it's meant to hold only almost-permanent `ExtensionActions` (extensions that have popups) that are our main brand calls-to-actions. Of course it all changes if the design isn't as logical or separate as this. 

But since the onion button is page-specific, I would suggest it goes in to the appropriate page actions area (or location bar) and keep `BraveActionsContainer` for the more permanent items. The visual design seems to agree with that via the visual separator.

Reply:
addressed here 9febd2beadb69a73ba9949387388fda66e01d5f0